keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_presets.py

```diff
@@ -22,6 +22,9 @@
                 "[1, 3, 3, 1] depths where the batch normalization "
                 "and SiLU activation are applied after the convolution layers."
             ),
+            "params": 2380416,
+            "official_name": "CSP_DARK_NET",
```

Comment:
The official name is just what we want to display for the model. Maybe camel case would look nicer: `"CSPDarkNet"`

Reply:
Done.